ballerina/encrypt_decrypt.bal

```diff
@@ -260,7 +260,23 @@ public isolated function encryptPgp(byte[] plainText, string publicKeyPath, *Opt
     'class: "io.ballerina.stdlib.crypto.nativeimpl.Encrypt"
 } external;
 
+# Writes the PGP-encrypted value of the given data to a file specified by the output file path.
```

Comment:
```suggestion
# Writes the PGP-encrypted value of the given input file to a file specified by the output file path.
```
It's better to say the nput is also a file. WDYT?

Reply:
+1, will change

Reply:
Fixed via [5ba88d7](https://github.com/ballerina-platform/module-ballerina-crypto/pull/567/commits/5ba88d7d21748a54d11ba4f7eda224128b2b288c)